src/Core/src/Core/IHandlerBehaviors.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Microsoft.Maui;
+
+// TODO .NET9/10 Make this public at some point. 
```

Comment:
The other APIs seem to be public, but this is not? Is this just because it is for the "interop" interface and the others are useful things? Not saying this _should_ be public, as we might not want to sign off on this API just yet.

Reply:
yea, we almost definitely want to make this public, but, I wanted to get this merged first and maybe we'll get this figured out before I merge. I just didn't want this to hold anything up as I work on this.